backend/de.metas.business/src/main/java-legacy/org/compiere/model/MProduct.java

```diff
@@ -335,9 +342,9 @@ private void createOrUpdateProductPlanningsForSelector()
 		}
 		
 		final ProductId productId = ProductId.ofRepoId(getM_Product_ID());
+		final OrgId orgId = OrgId.ofRepoId(getAD_Org_ID());
 
-		final IProductPlanningSchemaBL productPlanningSchemaBL = Services.get(IProductPlanningSchemaBL.class);
-		productPlanningSchemaBL.createOrUpdateProductPlanningsForSelector(productId, productPlanningSchemaSelector);
+		productPlanningSchemaBL.createOrUpdateProductPlanningsForSelector(productId, orgId, productPlanningSchemaSelector);
```

Comment:
i would suggest to extract that code in an model interceptor.

As a side benefit we will not wire"private final IProductPlanningSchemaBL productPlanningSchemaBL"  service each time a product is loaded.

Reply:
This code is called only once after the product was created, afaiu from 

```java
protected boolean afterSave(boolean newRecord, boolean success) {
[...]
	if(newRecord)
		{
			createOrUpdateProductPlanningsForSelector();
		}
```
so isnt' this already in Product's model interceptor?

Reply:
done